src/plugins/data/kibana.json

```diff
@@ -10,9 +10,10 @@
   "optionalPlugins": ["usageCollection"],
   "extraPublicDirs": ["common", "common/utils/abort_utils"],
   "requiredBundles": [
-    "usageCollection",
-    "kibanaUtils",
+    "embeddable",
```

Comment:
d'oh. so I introduced some dependencies on embeddable types in the data plugin in this PR, only to realize that (of course) embeddable has some type dependencies on data, thus creating another circular issue.

In particular what makes this tricky is that the actions that have been moved to data (value click and select range) rely on their context containing, among other things, `IEmbeddable`. This design is generally just confusing, but so far there are two ways I've thought of to mitigate this:

1. Define the _types_ for the action context in embeddable (since it already has type dependencies on data), and keep the actions themselves in data. The way the global mapping magically works means that the actions which are in data shouldn't need to import the types from embeddable directly.
2. Move the actions entirely to embeddable, which would require embeddable importing some stateful utilities from `data` which are currently internal.

The problem with option 1 is that it risks creating confusion having actions live in one place but the types for them defined in another.

The problem with option 2 is that we are now asking embeddable to know about stuff that clearly belongs in the data domain, and we need to increase the size of the data api to accommodate this change

Option 1 feels like the lesser of two evils -- but overall I don't know enough about uiActions to understand why embeddables need to be included in some of these action contexts in the first place; they don't appear to be using embeddables for anything internally, just passing them through. This feels like a design smell that we should revisit, but that's outside the scope of this PR.

Open to any other ideas I may have overlooked here...